<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1251</width>
      <height>791</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>1211</width>
       <height>191</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1209</width>
        <height>189</height>
       </rect>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>111</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>4</number>
        </property>
        <property name="verticalSpacing">
         <number>2</number>
        </property>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkOther">
          <property name="text">
           <string>OTHER</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkOpencv">
          <property name="text">
           <string>OPENCV</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkTiny">
          <property name="text">
           <string>TINY-FACE</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkMtcnn">
          <property name="text">
           <string>MTCNN</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>58</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>算法选择:</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>0</y>
         <width>191</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>已处理</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="fpNumber"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>设定频率</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="dealNumber"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>设置图片大小</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="wh"/>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>px</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="3">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>张</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>58</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>参数设置:</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>0</y>
         <width>267</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="horizontalSpacing">
         <number>5</number>
        </property>
        <property name="verticalSpacing">
         <number>2</number>
        </property>
        <item row="0" column="2">
         <widget class="QPushButton" name="nextButton">
          <property name="text">
           <string>下一张</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="stopButton">
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="startButton">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="splitBuntton">
          <property name="text">
           <string>手动裁剪</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>原始图像</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>裁剪后的图像</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="srcImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>751</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="resultImage">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>50</y>
       <width>441</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
